app/models/import/technology_builder.rb

```diff
@@ -35,15 +35,13 @@ def self.attribute(key)
     def self.build(key, data)
       units = NumberOfUnitsAttribute.call(data).round
       tech  = Technology.by_key(key)
-      attrs = {
-        'type'                        => key,
-        'name'                        => tech.name,
-        'units'                       => units,
-        'carrier'                     => tech.carrier,
-        'composite'                   => tech.composite,
-        'capacity'                    => tech.default_capacity,
-        'position_relative_to_buffer' => tech.default_position_relative_to_buffer
-      }
+      attrs = tech.defaults.merge(
+        'type'       => key,
+        'name'       => I18n.t("inputs.#{ key }"),
```

Comment:
Since the name is a presentation detail and now stored in a locale file, perhaps it's time to stop storing this attribute in the database altogether? The user can't customise it, so it doesn't have much purpose to exist anymore.